src/types/types.d.ts

```diff
@@ -20,11 +20,30 @@ export interface ButtonProps {
 }
 
 export interface ISetKeyPair {
-  setSecret: (secret: string | null) => void;
+  setSecretKey: (secret: string | null) => void;
   setPublicKey: (publicKey: string | null) => void;
 }
 
 export interface Ikeypair {
   publicKey: string;
   privateKey: string;
 }
+
+export interface Imodal {
```

Comment:
`IModal`

Reply:
Issue fixed in commit d23730ea5a20ee7c717bab26107db18ccebe94fd.